data-node/src/main/java/org/graylog/datanode/opensearch/configuration/OpensearchConfiguration.java

```diff
@@ -92,7 +93,18 @@ private String toValuesList(List<String> values) {
 
     public Environment getEnv() {
         final Environment env = new Environment(System.getenv());
-        env.put("OPENSEARCH_JAVA_OPTS", "-Xms%s -Xmx%s".formatted(opensearchSecurityConfiguration.getOpensearchHeap(), opensearchSecurityConfiguration.getOpensearchHeap()));
+
+        List<String> javaOpts = new LinkedList<>();
+        javaOpts.add("-Xms%s".formatted(opensearchSecurityConfiguration.getOpensearchHeap()));
+        javaOpts.add("-Xmx%s".formatted(opensearchSecurityConfiguration.getOpensearchHeap()));
+
+        opensearchSecurityConfiguration.getTruststore().ifPresent(truststore -> {
+            javaOpts.add("-Djavax.net.ssl.trustStore=" + truststore.location().toAbsolutePath());
```

Comment:
without having tested it, I am not sure if OS will be able to access e.g. a s3 bucket for data tiering if overriding the default trust store

Reply:
Good point, let me see if we can merge the default truststore with our and provide a complete set of certificates to the opensearch.